bitcoin/src/bip32.rs

```diff
@@ -209,6 +209,10 @@ impl FromStr for ChildNumber {
     }
 }
 
+impl AsRef<[ChildNumber] for ChildNumber{
+    fn as_ref(&self) -> &[ChildNumber] { &self.0 }
```

Comment:
[`core::ptr::slice_from_raw_parts`](https://doc.rust-lang.org/stable/core/ptr/fn.slice_from_raw_parts.html) should be used to convert the value into a 1 size slice (requires unsafe).

Unless there's another way to do this I'm missing...

Reply:
[`core::slice::from_ref`](https://doc.rust-lang.org/stable/core/slice/fn.from_ref.html)